.mlc_config.json

```diff
@@ -1,4 +1,9 @@
 {
+    "ignorePatterns": [
+		{
+			"pattern": "^https://pi-hole.net"
```

Comment:
Technically if this is a regex is should probably be `^https\:\/\/pi-hole\.net`

Reply:
Cool, thanks! I saw some errors in VSCode related to having the escape in front of the `:` and `.`. Do you know if this matters? It stopped being mad at me if the regex is `^https:\/\/pi-hole.net`.